mailet/standard/src/main/java/org/apache/james/transport/mailets/OnlyText.java

```diff
@@ -213,106 +213,101 @@ private void initEntityTable() {
-        charMap.put("auml", "\u00E4");
-        charMap.put("aring", "\u00E5");
-        charMap.put("aelig", "\u00E6");
-        charMap.put("ccedil", "\u00E7");
-        charMap.put("egrave", "\u00E8");
-        charMap.put("eacute", "\u00E9");
-        charMap.put("ecirc", "\u00EA");
-        charMap.put("euml", "\u00EB");
-        charMap.put("igrave", "\u00EC");
-        charMap.put("iacute", "\u00ED");
-        charMap.put("icirc", "\u00EE");
-        charMap.put("iuml", "\u00EF");
-
-        charMap.put("eth", "\u00F0");
-        charMap.put("ntilde", "\u00F1");
-        charMap.put("ograve", "\u00F2");
-        charMap.put("oacute", "\u00F3");
-        charMap.put("ocirc", "\u00F4");
-        charMap.put("otilde", "\u00F5");
-        charMap.put("ouml", "\u00F6");
-        charMap.put("divid", "\u00F7");
-        charMap.put("oslash", "\u00F8");
-        charMap.put("ugrave", "\u00F9");
-        charMap.put("uacute", "\u00FA");
-        charMap.put("ucirc", "\u00FB");
-        charMap.put("uuml", "\u00FC");
-        charMap.put("yacute", "\u00FD");
-        charMap.put("thorn", "\u00FE");
-        charMap.put("yuml", "\u00FF");
-        charMap.put("euro", "\u0080");
+        charMap.put("Ouml", "Ö");
+        charMap.put("Oacute", "Ó");
+        charMap.put("iquest", "¿");
+        charMap.put("yuml", "ÿ");
+        charMap.put("cent", "¢");
+        charMap.put("deg", "°");
+        charMap.put("aacute", "á");
+        charMap.put("uuml", "ü");
+        charMap.put("Otilde", "Õ");
+        charMap.put("Iacute", "Í");
+        charMap.put("frac12", "½");
+        charMap.put("atilde", "ã");
+        charMap.put("ordf", "ª");
+        charMap.put("sup2", "²");
+        charMap.put("sup3", "³");
+        charMap.put("frac14", "¼");
+        charMap.put("ucirc", "û");
+        charMap.put("brvbar", "¦");
+        charMap.put("reg", "®");
+        charMap.put("sup1", "¹");
+        charMap.put("THORN", "Þ");
+        charMap.put("ordm", "º");
+        charMap.put("eth", "ð");
+        charMap.put("Acirc", "Â");
+        charMap.put("aring", "å");
+        charMap.put("Uacute", "Ú");
+        charMap.put("oslash", "ø");
+        charMap.put("eacute", "é");
+        charMap.put("agrave", "à");
+        charMap.put("Ecirc", "Ê");
+        charMap.put("laquo", "«");
+        charMap.put("Igrave", "Ì");
+        charMap.put("Agrave", "À");
+        charMap.put("macr", "¯");
+        charMap.put("Ucirc", "Û");
+        charMap.put("igrave", "ì");
+        charMap.put("ouml", "ö");
+        charMap.put("iexcl", "¡");
+        charMap.put("otilde", "õ");
+        charMap.put("ugrave", "ù");
+        charMap.put("Aring", "Å");
+        charMap.put("Ograve", "Ò");
+        charMap.put("Ugrave", "Ù");
+        charMap.put("ograve", "ò");
+        charMap.put("acute", "´");
+        charMap.put("ecirc", "ê");
+        charMap.put("euro", "€");
+        charMap.put("uacute", "ú");
+        charMap.put("shy", "\\u00AD");
```

Comment:
\\u00AD does not have unicode value?

Reply:
it's an invisible space, not a great think to have in a source file